consensus/hotstuff/signature/randombeacon_inspector.go

```diff
@@ -16,8 +15,11 @@ type randomBeaconInspector struct {
 }
 
 // NewRandomBeaconInspector instantiates a new randomBeaconInspector.
-//
-// It errors with engine.InvalidInputError if any input is not valid.
+// The constructor errors if:
```

Comment:
I got the impression that the constructor only errors with `crypto.invalidInputsError` (?) I tried to only expose package-specific errors, which this error is not. So I would like to:
1. Either wrap it into a package-specific error if it is an expected error during normal operations. 
2. Or view a potential `crypto.invalidInputsError`  as in unexpected error. In this case, we would not need to expose the underlying error type, because we usually don't document the types of unexpected errors (only errors expected during normal operations).

Reply:
Maybe, I'll wrap it as a `model.ConfigurationError` 🤷